src/rantanplan/core.py

```diff
@@ -500,32 +516,41 @@ def get_word_stress(word, pos, tag, alternative_syllabification=False):
                      or (pos in ("PRON", "DET")
                          and tag.get("PronType", None) in (
                                  "Exc", "Int", "Dem"))
-                     or "".join(word).lower() in POSSESSIVE_PRON)):
+                     or "".join(
+                            word).lower() in STRESSED_PRON) and word_lower not in UNSTRESSED_FORMS):
```

Comment:
That's a weird way of breaking the line, isnt?

Reply:
Woopsie, will fix